routes/index.js

```diff
@@ -236,3 +193,84 @@ var checkAuth = function (req, res, next) {
     return next();
   res.redirect('/');
 }
+
+
+/*getStnMngmnt will render a table of all stations that match filter parameter.
+  *    If filter == "registered", only registered stations will be displayed (i.e. registered = true)
+  *    If filter == "unregistered", only unregistered stations will be displayed (i.e. registered = false)
+  *    Otherwise, all stations will be displayed sorted with unregistered stations first.
+  *    If "messageType" and "message" are defined in req, it will trigger alert message passing in DOM.
+  *********************************************************************************************************/
+function getStnMngmnt(req, res) {
+  //set filter based on query parameter passed in
+  var filter = req.params.filter;
+  filter = (filter === "registered") ? true : (filter === "unregistered") ? false : undefined;
+
```

Comment:
If req.messageType exists, then this function was called via "next" by some previous route and it passed in a message that we should pass on to the DOM. If not, the data elements passed in will be undefined and do nothing.